termennetwerk.ts

```diff
@@ -51,22 +52,33 @@ export async function getUriOfSearchTerm(sources: string[], searchTerm: string):
     const cacheKey = searchTerm.toLowerCase();
     if (cache[cacheKey]) return cache[cacheKey];
     const response: GraphQlResponse = await client.request(graphQlQuery, { sources, query: searchTerm });
-    if (!Array.isArray(response.terms) ||
-        response.terms[0].result.terms.length !== 1 ||
-        !response.terms[0].result ||
-        !response.terms[0].result.terms ||
-        !Array.isArray(response.terms[0].result.terms)) {
+    if (!Array.isArray(response.terms[0].result.terms) ||   // terms[] does not exists when an error occurs like a timeout
```

Comment:
To make the code a bit easier on the eyes, you could first create a separate variable that holds the result:

`const result = response.terms[0].result;`